products/memgraph/v0.51.0/how_to_guides/use-query-modules-provided-by-memgraph.md

```diff
@@ -51,17 +51,182 @@ loaded modules:
 CALL mg.reload_all();
 ```
 
-### Graph Algorithms as Query Modules [Enterprise]
+### Community Graph Algorithms as Query Modules
+
+Memgraph Community edition comes with a set of Python query modules based on
+[NetworkX](https://networkx.github.io/) library of algorithms. Modules are
+already packaged within all Memgraph packages, but `NetworkX` might have to
+be installed by running the following command:
+
+```
+pip3 install networkx
+```
+
+NOTE: The following "How to Guides" provide explanation of basic usage. To find
+out more details about each module and documentation of each procedure, please
+take a look at installed Python files. Under Linux, the files should be located
+under `/usr/lib/memgraph/query_modules`.
+
+#### Graph Analyzer
+
+The purpose of this module is to get more insights about the stored graph. To
+illustrate functionality the following graph might be used.
```

Comment:
...the following graph *will* be used:

Reply:
Use `:` instead of `.` at the end of the sentence because you have 'the following'